util/net.go

```diff
@@ -0,0 +1,35 @@
+package netutils
```

Comment:
for one we should call the directories containing the packages the same name as the package. this can be useful regarding tooling and human expectations, too. also a custom in go.

on the other hand, i kinda dislike package names like 'util', 'common', etc. Can we call this simply 'net'?

on the third hand, this functionality feels kind of a triviality, or at least a bordercase. do we expect it to be globally useful? do we expect this package to grow with time? is anyone already aware of stuff that could go here during a refactoring?